File: PspDataPresenter.Core/Models/VotingEndModel.cs
using System;
using System.Collections.Generic;

namespace PspDataPresenter.Core
{
    public class VotingEndModel
    {
        public int Id { get; set; }
        public RoomModel Room { get; set; }
        public int VotingNo { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public VotingSummaryModel Summary { get; set; }
        public List<MemberVoteModel> MemberVotes { get; set; }
        public DateTime Timestamp { get; set; }
    }
}

File: PspDataPresenter.App/data-presenter-app/src/app/_core/models/voting-end.model.ts
import { MemberVoteModel } from "./member-vote.model";
import { RoomModel } from "./room.model";
import { VotingSummaryModel } from "./voting-summary.model";

export interface VotingEndModel
{
    id: number;
    room: RoomModel;
    votingNo: number;
    start: Date | string;
    end: Date | string;   
    votingSummary: VotingSummaryModel;
    memberVotes: MemberVoteModel[];
    timestamp: Date | string;
}